### DISCUSSION TITLE
[Cinnamon] Move window list button to second display

### DISCUSSION THREAD

**Discussion by kr4ssi** [linuxmint/discussions#461] [Ideas]
I have a setup by 2 displays with a window list applet on each showing only windows of the correspondent display.

I know that if I move a window to the other display, the window list button will get moved too, but I would like, if that would be possible with the window list button itself too. 
So I would just to have to grab the button instead of the title bar of the window to move it to the window list applet of the second display.

**Comment by klangman** [comment]
Sounds like a neat idea.. 

I'll work on adding this to my window-list (CassiaWindowList, https://cinnamon-spices.linuxmint.com/applets/view/372) this week. I won't require a drop in the other monitors window-list, rather a drop anywhere on a monitor other than the monitor that the window currently resides in will move the window to the new monitor.   

I'll post another note here once the feature has been added to cinnamon-spices so you can give it a shot if you feel like it. It will show up on my github in a day or two if you are really keen on trying it out: https://github.com/klangman/CassiaWindowList

**Comment by klangman** [comment]
I just updated CassiaWindowList to version 2.2.0 which added support to drag-and-drop a window-list button on to the desktop, this will move the window to the monitor and the workspace that the button was dropped on and then give the window the focus. I believe this is the feature you were asking for?

To get this feature you would need to switch from the default window-list applet to CassiaWindowList using the "Applets" tool in Cinnamon system settings. Hopefully CassiaWindowList works well for you, if not you can always go back to the default window-list applet.
